src/lib/utils.ts

```diff
@@ -310,6 +318,19 @@ export function parseRadicleId(id: string): string {
   }
 }
 
+// Checks that the input is an object with a `reason` property.
+// Is used to check if ether.js errors have a `reason` property.
+export function hasReasonProperty(
```

Comment:
Were using this only in one place, could we inline it for now?

Reply:
1e4ebf0